storm-core/src/clj/backtype/storm/converter.clj

```diff
@@ -23,7 +25,9 @@
       (if (.get_used_ports supervisor-info) (into [] (.get_used_ports supervisor-info)))
       (if (.get_meta supervisor-info) (into [] (.get_meta supervisor-info)))
       (if (.get_scheduler_meta supervisor-info) (into {} (.get_scheduler_meta supervisor-info)))
-      (.get_uptime_secs supervisor-info))))
+      (.get_uptime_secs supervisor-info)
+      (.get_version supervisor-info);;log
```

Comment:
Please clean up the comment, not sure what ;;log means, and move the closing ')' to this line